example/test.py

```diff
@@ -55,26 +81,24 @@ def test_expected_processes(self, web_only_container):
 
         assert_that(ps_data, HasLength(5))
```

Comment:
Do we need this length check? The next two lines pop and assert pid 1, then assert that the remaining entries match what we expect. That should be sufficient, unless there's some kind of issue with `MatchesSetwise` and duplicate entries.